### ISSUE TITLE
Support for React 16

### ISSUE THREAD

**Issue by msmfsd** [tomchentw/react-google-maps#677]
Upgrading react and react-dom to `v16.0.0` gets following error:

```
Module not found: Error: Can't resolve 'react/lib/React' in '/app/node_modules/react-google-maps/node_modules/react-dom/lib'
```

**Comment by tomchentw** [comment]
@msmfsd which module throws this error?

**Comment by msmfsd** [comment]
@tomchentw I get it with `react-google-maps@4.11.0`

**Comment by msmfsd** [comment]
its a webpack compile

**Comment by tomchentw** [comment]
@msmfsd `4.11.0` is old and not supported anymore. Please try upgrading to latest version:
https://github.com/tomchentw/react-google-maps/releases/tag/v9.0.3

You can find changelogs here:
https://github.com/tomchentw/react-google-maps/blob/master/CHANGELOG.md

Thanks!

**Comment by tomchentw** [comment]
Released [v9.1.0](https://github.com/tomchentw/react-google-maps/releases/tag/v9.1.0)
